docs/config.md

```diff
@@ -40,6 +40,8 @@ DataSource:
     mySQLColumnEmail: 'email'
     # Column for storing if a player is logged in or not
     mySQLColumnLogged: 'isLogged'
+    # Column for storing if a player has a valid session or not
+    mySQLColumnHasSession: 'isLogged'
```

Comment:
Intentional to have the same name as `mySQLColumnLogged`? Maybe have an explanation for that?

Reply:
LOL, i shoudn't code on the train.